packages/mux-player/src/index.ts

```diff
@@ -510,9 +537,54 @@ class MuxPlayerElement extends VideoApiElement {
     };
   }
 
-  set tokens(val) {
+  /**
+   * Set the signing tokens for the Mux asset URL's.
+   */
+  set tokens(val: Tokens | undefined) {
     this.#tokens = val ?? {};
   }
+
+  /**
+   * Get the playback token for signing the src URL.
+   */
+  get playbackToken() {
```

Comment:
thought: what about interfacing with these tokens via `tokens()`? So, rather than
```js
player.playbackToken = '123'
```
you would
```js
player.tokens.playback = '123'
```

Reply:
good call, the setting like that does work at the moment. only downside is that it would not trigger the attributeChangedCallback. thinking of this, even setting the full tokens property would not re-render. maybe it should

Reply:
internally, we could make that when we set `tokens.playback = '123'` we call `setAttribute`. Not sure if that's worth doing over having `player.playbackToken = '123'`

Reply:
I will get kinda messy with setters in the plain object etc or we would need to create a new class for Tokens.
maybe lets iterate on it and see if users prefer to use it like that.